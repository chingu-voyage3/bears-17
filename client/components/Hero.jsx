import React from 'react';
import { Link } from 'react-router-dom';

const Hero = () => (
  <section className="hero">
    <div className="wrapper wrapper--flex wrapper--flex-wrap">
      <div className="hero__description">
        <h1 className="hero__title">Sharpen Your Interviewing Skills</h1>
        <h2 className="hero__subtitle">... And Get the Job You Want</h2>
      </div>
      <div className="hero__cta">
        <Link to="/register"className="btn btn--lg btn--success">
          Register and Start Asking Questions
        </Link>
        <p>... or just...</p>
        <Link to="/questions" className="btn btn--sm btn--hollow">
          Browse the Latest Questions
        </Link>
      </div>
    </div>
  </section>
);

export default Hero;
