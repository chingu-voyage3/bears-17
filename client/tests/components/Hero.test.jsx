import React from 'react';
import renderer from 'react-test-renderer';

import Hero from 'Components/Hero';

describe('Hero', () => {
  it('should match the snapshot', () => {
    const hero = renderer.create(<Hero />);

    expect(hero.toJSON()).toMatchSnapshot();
  });
});
