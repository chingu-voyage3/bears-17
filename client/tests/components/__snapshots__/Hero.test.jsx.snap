// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero should match the snapshot 1`] = `
<section
  className="hero"
>
  <div
    className="wrapper wrapper--flex wrapper--flex-wrap"
  >
    <div
      className="hero__description"
    >
      <h1
        className="hero__title"
      >
        Sharpen Your Interviewing Skills
      </h1>
      <h2
        className="hero__subtitle"
      >
        ... And Get the Job You Want
      </h2>
    </div>
    <div
      className="hero__cta"
    >
      <button
        className="btn btn--lg btn--success"
      >
        Click Here to Ask a Question
      </button>
      <p>
        ... or just...
      </p>
      <button
        className="btn btn--sm btn--hollow"
      >
        Browse the Most Popular Answers
      </button>
    </div>
  </div>
</section>
`;
